src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private int randomLimit = 11;
```

Comment:
```suggestion
    private static final int RANDOM_LIMIT = 11;
```
тут доцільно використати константу 
https://stackoverflow.com/a/66076

Reply:
Дякую за посилання!